src/components/banner/banner.njk

```diff
@@ -10,8 +10,8 @@
       </button>
       </div>
     </header>
-    <div class="usa-banner-content usa-grid usa-accordion-content" id="gov-banner">
-      <div class="usa-banner-guidance-gov usa-width-one-half">
+    <div class="usa-banner-content usa-accordion-content grid-container grid-row" id="gov-banner">
```

Comment:
Would have liked to be able to add `grid-gap` but it messes up the margins - I'd prefer not to have to add another div.

Reply:
cc @thisisdano

Reply:
Addressed in https://github.com/uswds/uswds/pull/2757/